crates/goose-cli/src/commands/configure.rs

```diff
@@ -158,7 +161,8 @@ pub fn get_recommended_model(provider_name: &str) -> &str {
         "databricks" => "claude-3-5-sonnet-2",
         "ollama" => OLLAMA_MODEL,
         "anthropic" => "claude-3-5-sonnet-2",
-        "google" => "gemini-1.5-flash",
+        "google" => GOOGLE_DEFAULT_MODEL,
```

Comment:
nit: this seems like something we should standardize in the providers for openai, databricks, and anthropic

Reply:
done